PF/Assignment2/assignment_1.java

```diff
@@ -0,0 +1,72 @@
+public class assignment_1 
+{
+public static void main(String args[])
+   {
+	assignment_1 as1=new assignment_1();
+	int row=9,n=5;
+	String pyramid[]=new String[row];
+	pyramid=as1.printPyramid(row, n);
+	for(int i=0;i<row;i++)
+	{
+		System.out.print(pyramid[i]);
+	    System.out.println();
+	}
+	
+   }
+String spaces(int row,int n)
```

Comment:
ACTION: the argument 'n' is nowhere used in the function. Looking at the code I think it does not work for 'n' rather it can print triangle till 5 only. You need to rewrite it so that it can print up to 'n'.